lib/tasks/data.rake

```diff
@@ -202,20 +225,39 @@ namespace :aleftng do
           #puts "QUESTION EXISTS"
         end
         import_concepts(concept_names, lo) if concept_names
+
+        # Pridanie obrázov do prisluchajuceho learning objectu
+        if picture
+          picture = picture.split('/').last
+          #puts "PICTURE QALO: #{picture}"
+
+          if picture_names.include? picture
+            index = picture_names.index(picture)
+            image = File.read(picture_paths[index])
```

Comment:
Toto je velmi ne-ruby way, pozriet sa, ci to existuje v poli, vyhladat si index v tom poli a ten pouzit v dalsom inom poli. To uz by to rovno mohol byt hash.

Resp. preco to vobec funguje takto, ze si zvlast vylistujes nazvy v adresari do pola, zvlast cesty v adresari do pola a potom tam hladas indexy, nestaci k nazvu obrazku prilepit priponu a to rovno nacitat (resp. pripadne spravit `File.exists?` ked uz nie chytat vynimku)?